test_files/partial/partial.js

```diff
@@ -1,4 +1,3 @@
-// test_files/partial/partial.ts(7,1): warning TS0: omitting heritage reference to a type literal: Partial<Base>
```

Comment:
Shouldn't this still do something with `Partial<Base>`? I think this is one of the `debugWarn` cases I commented on, isn't it?